help/CreateImageAndAzureResources.md

```diff
@@ -124,3 +124,22 @@ Generated tool versions and details can be found in related projects:
 - [Go](https://github.com/actions/go-versions)
 - [Node](https://github.com/actions/node-versions)
 - [Boost](https://github.com/actions/boost-versions)
+
+### Post-generation scripts
+Scripts for setting up some specific configuration after packer build procedure are placed in `post-generation` folder. These scripts are intended to apply configuration which cannot be covered during packer build procedure. Scripts are required, since packer build and default agent users are not the same and some specific configuration should be applied after build process under agent's user.
+
+Detailed description for the scripts:
```

Comment:
I think we should add info where these scripts are located on machine (they are copied to machine during image generation) and may be add simple command how to run all of them.

Reply:
Thank you for the comments, I'll definitely add this information shortly.